utils/test_fv3gfs_dataset.py

```diff
@@ -1,8 +1,14 @@
+import copy
 import os
+import numpy as np
```

Comment:
import order?

Reply:
Unless/until we have a CI linter that actually enforces some standard, IMO it's not worth worrying about these details.